Makefile

```diff
@@ -37,7 +37,11 @@ all-modules:
 	@echo $(ALL_MODULES) | tr ' ' '\n' | sort
 
 .PHONY: all
-all: common gotest otelcontribcol otelcontribcol-unstable
+all: all-common gotest otelcontribcol otelcontribcol-unstable
```

Comment:
what is the rule between common and non-common? Should test be there as well?

Reply:
I see your point. It's not very consistent. I will write up a new issue to proposal guidelines on how targets should be scoped.